templates/web/greenwich/header_logo.html

```diff
@@ -1,18 +1,35 @@
-    <a href="http://www.royalgreenwich.gov.uk/" id="site-logo">[% site_name %]</a>
+<a href="http://www.royalgreenwich.gov.uk/" id="site-logo">[% site_name %]</a>
+<ul id="greenwich-menu-desktop">
+    <li><a href="http://www.royalgreenwich.gov.uk/pay">Make a payment</a></li>
+    <li><a href="http://www.royalgreenwich.gov.uk/apply">Apply for it</a></li>
+    <li><a href="/">Report a problem</a></li>
+    <li><a href="http://www.royalgreenwich.gov.uk/contact">Contact us</a></li>
+</ul>
 </div>
-<div class="container main-menu-container">
-    <ul id="greenwich-menu-desktop">
+
+<div class="main-menu-container">
```

Comment:
This (or one of its children) needs some horizontal padding, to prevent this happening at screen widths between 47–82em:

![Screenshot_2020-12-04 FixMyStreet](https://user-images.githubusercontent.com/739624/101145536-d58c0780-3611-11eb-8624-d1e524f7bae9.png)